code.user.js

```diff
@@ -3071,7 +3081,9 @@
                 }
             }
 
-            $('#my_market_selllistings_number').append(`<span id="my_market_sellistings_total_price">, ${formatPrice(totalPriceBuyer)} ➤ ${formatPrice(totalPriceSeller)}</span>`);
+            $('#my_market_selllistings_number')
+            .append(`<span id="my_market_sellistings_total_amount"> [${totalAmount}]</span>`)
+            .append(`<span id="my_market_sellistings_total_price">, ${formatPrice(totalPriceBuyer)} ➤ ${formatPrice(totalPriceSeller)}</span>`);
```

Comment:
Please keep original append on first line and then the next can be on the other one aligned with the previous append statement.

Reply:
@Nuklon, did I understand you correctly?
Commit 805469b

Reply:
Yes, thanks.